### PR TITLE
[TEVA-2025] Update "personal statement" page

### PR DESCRIPTION
**Pull request by cesidio**
## Jira ticket URL
https://dfedigital.atlassian.net/browse/TEVA-2025

## Screenshots of UI changes:
Screenshot 2021-03-02 at 17 40 27

### CODE REVIEW

File: app/views/jobseekers/job_applications/build/personal_statement.html.slim [hunk 1]
```diff
@@ -8,6 +8,20 @@
     h1.govuk-heading-l = t(".heading")
     p.govuk-body = t(".description")
 
+    - if job_application.vacancy.documents.any?
+      .govuk-notification-banner aria-labelledby="govuk-notification-banner-title" data-module="govuk-notification-banner" role="region"
```
Comment (cpjmcquillan) [inline app/views/jobseekers/job_applications/build/personal_statement.html.slim:12]: we should be able to use `govuk_notification_component` (or it's called something like that)

File: config/locales/jobseekers.yml [hunk 2]
```diff
@@ -85,9 +85,12 @@ en:
           heading: Personal details
           title: Personal details — Application
         personal_statement:
+          additional_instructions: Additional instructions from the school hiring staff
+          banner_title: Job listing and supporting documents
+          banner_description_html: Documents supplied with `%{job_link}` listing that may be of some help with your personal statement.
```
Comment (david-mears-dfe) [inline config/locales/jobseekers.yml:90]: Please use these nice curly quotes: 
Left ‘
Right ’

These are the ones used here:
Screenshot 2021-03-02 at 17 58 39

**Comment by martin-bangoura** [comment]
Review app deployed to <https://teaching-vacancies-review-pr-2995.london.cloudapps.digital>

File: app/views/jobseekers/job_applications/build/personal_statement.html.slim [hunk 3]
```diff
@@ -8,6 +8,18 @@
     h1.govuk-heading-l = t(".heading")
     p.govuk-body = t(".description")
 
+    - if job_application.vacancy.documents.any?
+      = govuk_notification_banner(title: t(".banner_title")) do |banner|
+        - banner.add_heading
```
Comment (cpjmcquillan) [inline app/views/jobseekers/job_applications/build/personal_statement.html.slim:13]: Maybe add a comment about this part

File: app/views/jobseekers/job_applications/build/personal_statement.html.slim [hunk 4]
```diff
@@ -8,6 +8,18 @@
...
+        = t(".banner_description_html", job_link: govuk_link_to(job_application.vacancy.job_title, job_path(job_application.vacancy), class: "document-link-component__link", target: "_blank"))
+        ul.hide-bullet-points
+          - job_application.vacancy.documents.in_groups_of(1, false).each do |row|
```
Comment (cpjmcquillan) [inline app/views/jobseekers/job_applications/build/personal_statement.html.slim:16]: In fact we might be able to just do 
```slim
= render Shared::DocumentLinkComponent.with_collection(job_application.vacancy.documents)
```
I'm not sure why I did it this other way before

**Review by cpjmcquillan — APPROVED** [review]

File: app/views/jobseekers/job_applications/build/personal_statement.html.slim [hunk 5]
```diff
@@ -8,6 +8,19 @@
     h1.govuk-heading-l = t(".heading")
     p.govuk-body = t(".description")
 
+    - if job_application.vacancy.documents.any?
+      = govuk_notification_banner(title: t(".banner_title")) do |banner|
+        / added the following line because we need at least one heading for the component to be rendered
+        - banner.add_heading
+        = t(".banner_description_html", job_link: govuk_link_to(job_application.vacancy.job_title, job_path(job_application.vacancy), class: "document-link-component__link", target: "_blank"))
```
Comment (cpjmcquillan) [inline app/views/jobseekers/job_applications/build/personal_statement.html.slim:15]: Do we need this class on the job title link?

Comment (cesidio) [reply app/views/jobseekers/job_applications/build/personal_statement.html.slim]: nope, removed

**Pull request merged**